### PR TITLE
Fix #5339: Do not lowercase resource extensions everywhere

### LINKED ISSUE #5339: Mimetype extension should be case preserving and is not
Filenames are:
* TEST.png
* TEST.PNG

Both are shown equally which is very confusing.
image

### PR DESCRIPTION
**Pull request by dschmidt**
... but use toLowerCase() where needed.

## Description
I've removed `toLowerCase()` from `_getFileExtension()`, searched through the codebase for `extension` and added `toLowerCase()` where applicable.

It super easy to miss occurances, so I would really like if someone else could check this out and see if I missed something.

Should we do a regression test of some sort?

## Related Issue

- Fixes #5339

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Screenshots (if appropriate):
Screenshot_20211208_224913

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] Add regression test

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingAccept</strong> https://drone.owncloud.com/owncloud/web/20768/16/1
The following scenarios passed on retry:
- webUISharingAcceptSharesToRoot/acceptShares.feature:108

**Comment by ownclouders** [comment]
Results for <strong>oC10XGAPortrait1</strong> https://drone.owncloud.com/owncloud/web/20768/46/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Review by kulmann — APPROVED** [review]
LGTM 👍

File: packages/web-app-media-viewer/src/App.vue [hunk 1]
```diff
@@ -208,7 +208,7 @@ export default {
     },
 
     isActiveMediaFileTypeVideo() {
-      return this.videoExtensions.includes(this.activeMediaFile.extension)
+      return this.videoExtensions.includes(this.activeMediaFile.extension.toLowerCase())
```
Comment (kulmann) [inline packages/web-app-media-viewer/src/App.vue:211]: Might be a low hanging fruit: the `activeMediaFile` should have a field `mimeType` by now. Would be much easier and less error prone to check if the mimetype references a video format. Then you could ditch this one `toLowerCase` case. 😁

**Comment by kulmann** [comment]
> It super easy to miss occurances, so I would really like if someone else could check this out and see if I missed something.
Didn't find additional ones, and the e2e test coverage is good so far.

> Should we do a regression test of some sort?
Yeah, maybe. But the code paths that you touched don't look like it would be a good idea there... so I'd be fine without additional tests.

File: packages/web-app-media-viewer/src/App.vue (hunk 1 above)
Comment (kulmann) [reply packages/web-app-media-viewer/src/App.vue:211]: let's get this PR merged and look into that in another PR ok?

Comment (dschmidt) [reply packages/web-app-media-viewer/src/App.vue:211]: Yep, was my gut feeling too!

Comment (dschmidt) [reply packages/web-app-media-viewer/src/App.vue:211]: It's ready from my side, go ahead and merge if you think so too!

**Pull request merged**